src/components/KPrompt/KPrompt.vue

```diff
@@ -7,29 +7,28 @@
     :title="displayTitle"
   >
     <template #header-content>
-      <div class="k-prompt-header w-100">
-        <div class="k-prompt-header-content d-flex align-items-center w-100">
+      <div class="k-prompt-header">
+        <div class="k-prompt-header-content">
           <slot name="header-content">
             <KIcon
               v-if="type === 'warning'"
-              class="mr-2"
-              color="var(--white)"
+              class="warning-icon"
+              :color="`var(--white, ${KUI_COLOR_TEXT_INVERSE})`"
               icon="warning"
-              secondary-color="var(--yellow-400)"
-              size="20"
+              secondary-color="currentColor"
```

Comment:
issue: I'm not sure this makes sense; this means the color of the parent would have to be yellow

Reply:
No, I set color for the `.warning-icon` class I've given this element